gradle/tasks/docker.gradle

```diff
@@ -0,0 +1,58 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.bmuschko:gradle-docker-plugin:${gradleDockerPluginVersion}"
+    }
+}
+
+apply plugin: com.bmuschko.gradle.docker.DockerRemoteApiPlugin
+
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+
+task dockerBuildImageWeb(type: DockerBuildImage) {
+    group 'Docker'
+    description 'Builds the basic web server-only Docker image'
+    // Change to 'dependsOn' when we get our up-to-date sorted
+    mustRunAfter allprojects.collect { it.tasks.matching { it.name.equals('tomcatDeploy') } }
+
+    dockerFile file('docker/Dockerfile')
+    inputDir rootProject.projectDir
+    tag 'apereo/uportal'
+
+}
+
+task dockerBuildImageCli(type: DockerBuildImage) {
+    group 'Docker'
+    description 'Builds the Docker image used to invoke Import/Export tasks from within a container'
+    // Change to 'dependsOn' when we get our up-to-date sorted
+    mustRunAfter allprojects.collect { it.tasks.matching { it.name.equals('tomcatDeploy') } }
+
+    dockerFile file('docker/Dockerfile-cli')
+    inputDir rootProject.projectDir
+    tag 'apereo/uportal-cli'
+
+}
+
+task dockerBuildImageQuickstart(type: DockerBuildImage) {
+    group 'Docker'
+    description 'Builds the uPortal Quickstart Docker image'
+    // Change to 'dependsOn' when we get our up-to-date sorted
+    mustRunAfter allprojects.collect { it.tasks.matching { it.name.equals('tomcatDeploy') } }
+    mustRunAfter dockerBuildImageCli
+
+    dockerFile file('docker/Dockerfile-quickstart')
+    inputDir rootProject.projectDir
+    tag 'apereo/uportal-quickstart'
```

Comment:
should this tag also include the uPortal version?

```gradle
tag "apereo/uportal-quickstart:${uPortalVersion}"
```

Reply:
Yep -- makes sense.